CHANGELOG.adoc

```diff
@@ -66,6 +66,7 @@ The subscription threads used to keep the application alive, but will now allow
 * Spanner array param binding
 * Events and Auditing support
 * Multi-Instance support (#1530)
+* Fixed conversion for timestamps older than unix epoch (#2043)
```

Comment:
Add #2053:  populate key fields when virtual key property is set to unblock the REST repository PUT operation.